Added a few changes (#159)
iden3/rollup: README.md

### 2.0. Initial considerations<a id="2_0"></a>

> >A **batch** is a rollup "block" 

Reviewer: ```suggestion
>A **batch** is a rollup block
```

Reviewer: not a huge fan of quotation marks around words. i think it's clear enough without

>An **operator** is a batch producer

> **rollup address** is the compressed format of the babyjubjub public key

> **Forging** refers to the creation of a batch (off-chain) and the subsequent (on-chain) verification of the attached zkSnark.

> Each batch takes around 45 seconds to 1 minute to be forged by an operator, but these batches can be forged in parallel. 

Reviewer: ```suggestion
Each batch takes around 45 seconds to 1 minute to be forged by an operator, but batches can be forged in parallel. 
```

>A **batch** is a rollup block

>An **operator** is a batch producer

> **rollup address** is the compressed format of the babyjubjub public key

> **Forging** refers to the creation of a batch (off-chain) and the subsequent (on-chain) verification of the attached zkSnark.

> Each batch takes around 45 seconds to 1 minute to be forged by an operator, but batches can be forged in parallel. 

Reviewer: ```suggestion
Each batch takes around 45 seconds to 1 minute to be forged by an operator. 

> note: while in the current implementation batches can't be forged in parallel, we plan to make it possible to parallelise this step.
```
